tests/test_ind_indygo_service.py

```diff
@@ -40,29 +40,26 @@ def test_description():
 
 
 def test_start():
-    assert parsed_items[0]["start"] == datetime(2023, 1, 19, 10, 0)
+    assert parsed_items[0]["start"] == datetime(2024, 1, 18, 10, 0)
 
 
 # def test_end():
 #     assert parsed_items[0]["end"] == datetime(2019, 1, 1, 0, 0)
```

Comment:
Hmmm, we need to get this assertion working (preferable!) or we should delete this commented out code.